apps/researcher/src/app/[locale]/objects/[id]/date-created-to-string.test.tsx

```diff
@@ -0,0 +1,57 @@
+import dateCreatedToString from './date-created-to-string';
+import {describe, expect, it} from '@jest/globals';
+
+describe('dateCreatedToString', () => {
```

Comment:
Perhaps also add a test with a startDate and/or endDate that use a full date notation (i.e. including time)? Then we can test whether `dateStyle: 'medium',` works.

Reply:
Good point!